ggml/src/ggml-cpu/llamafile/sgemm.cpp

```diff
@@ -1803,6 +2365,18 @@ bool llamafile_sgemm(int64_t m, int64_t n, int64_t k, const void *A, int64_t lda
             ith, nth};
         tb.matmul(m, n);
         return true;
+#elif defined(__MMA__)
+        if (n < 8 && n != 4)
+           return false;
+        if (m < 8 && m != 4)
+           return false;
+        tinyBLAS_Q0_PPC<block_q8_0, block_q8_0, float> tb{
+            k, (const block_q8_0 *)A, lda,
+            (const block_q8_0 *)B, ldb,
+            (float *)C, ldc,
+            ith, nth};
```

Comment:
with https://github.com/ggerganov/llama.cpp/pull/10714

You will have to take ith/nth from params->ith/params->ith. It may be the only change you need to do.

Reply:
I have made these changes and pushed. @slaren / @Djip007 / @ggerganov Please review the changes.